sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ext/opentelemetry_span/_schema.py

```diff
@@ -17,9 +17,7 @@ class OpenTelemetrySchemaVersion(
 
 class OpenTelemetrySchema:
 
-    SUPPORTED_VERSIONS = [
-        OpenTelemetrySchemaVersion.V1_19_0,
-    ]
+    SUPPORTED_VERSIONS = (OpenTelemetrySchemaVersion.V1_19_0,)
```

Comment:
Why we change the type?

Reply:
I remember hitting some mypy issues while using a list when I was testing things a while back. It doesn't seem to be a problem anymore, as I can't seem to recreate the issue. Would you prefer I keep this as a list even though we don't expect it to be mutated?